packages/flutter_tools/lib/src/commands/build_web.dart

```diff
@@ -0,0 +1,38 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import '../base/logger.dart';
+import '../build_info.dart';
+import '../globals.dart';
+import '../runner/flutter_command.dart' show ExitStatus, FlutterCommandResult;
+import '../web/compile.dart';
+import 'build.dart';
+
+class BuildWebCommand extends BuildSubCommand {
+  BuildWebCommand() {
+    usesTargetOption();
+    usesPubOption();
+    defaultBuildMode = BuildMode.release;
+  }
+
+  @override
+  final String name = 'web';
+
+  @override
+  bool get hidden => true;
+
+  @override
+  final String description = 'Build an Web application bundle.';
```

Comment:
Maybe indicate that this is experimental or something (I assume that's the reason it's hidden above)?

Reply:
Add all caps to make it scary